web_src/js/components/RepoActionView.vue

```diff
@@ -32,6 +32,7 @@
                 <ActionRunStatus :status="job.status"/>
                 <span class="ui text gt-mx-3">{{ job.name }}</span>
               </a>
+              <span class="step-summary-dur">{{ job.duration }}</span>
```

Comment:
```suggestion
              <span class="step-summary-duration">{{ job.duration }}</span>
```

No need for obscure abbreviations

Reply:
Thanks for your suggestion.
Done.